entrypoint.py

```diff
@@ -29,7 +29,10 @@
 with open(os.environ["GITHUB_ENV"], "a") as environment_file:
 
     for k, v in mergedConfig.items():
-        environment_file.write(f"{k.upper()}={v}\n")
+        if "\n" in str(v):
+            environment_file.write(f"{k.upper()}<<delimiter\n{v}delimiter\n")
```

Comment:
this is exactly the way I used the functionality and it failed to work. has this been tested with use cases?